drf_to_mkdoc/utils/schema.py

```
@@ -1,85 +1,205 @@
 
-        # If this is a ViewSet, determine the action name
-        if isinstance(view_instance, ViewSetMixin):
-            action = callback.actions.get(method.lower())
-            if action:
-                view_instance.action = action
-                view_instance.request = type("MockRequest", (), {"method": method.upper()})()
+    def _create_view_instance(self):
+        """Create view instance for introspection."""
+        try:
+            self.view_instance = self.view()
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)